site/guide/model-validation/_assess-compliance-link-evidence.qmd

```diff
@@ -17,3 +44,8 @@
 1. Click **Update Linked Evidence**.
 
 The newly linked-to evidence now gets shown under **Developer Evidence**.
+
+::::
+
```

Comment:
Technically, extra blank lines should make the Markdown linter in VS Code complain. (If you don't have a linter enabled, it might be helpful to start using one. Future PR checks might include linting our docs source.)

(There's a few of these throughout this PR.)

Reply:
Hm, that's a good point, but I find I absolutely cannot navigate all the nested `:::` without some extra spacing as a human! Does it really affect the rendering 🤔

Reply:
It's not an issue right now as we currently don't lint our Markdown source in CI/CD checks. 🙂